public/docs/_examples/toh-4/ts/app/hero.service.ts

```diff
@@ -12,8 +12,8 @@ export class HeroService {
   getHeroes() {
     return Promise.resolve(HEROES);
   }
-  // #enddocregion get-heroes
-  // #enddocregion just-get-heroes
+  // #enddocregion get-heroes, just-get-heroes
+  // #enddocregion
```

Comment:
I've always assumed that we could not specify the unnamed docregion when other docregions are being named. But I could be wrong (I'll test that out some time).